duckduckgo/iOS
# text distribution across lines

Reviewer: Mia or Caine
Asana:

https://app.asana.com/0/414235014887631/489057161850488
https://app.asana.com/0/414235014887631/482056236481737

CC:

**Description**:

Split text across lines for better aesthetics. 

**Steps to test this PR**:
1. View overview while gathering message is displayed and observe message split across lines
1. View privacy practices  and observe message is split across lines
1. View privacy practices  and observe TOSDR ends with a period
